qa/L0_trace/opentelemetry_unittest.py

```diff
@@ -377,16 +377,21 @@ def _test_trace(
         """
         time.sleep(COLLECTOR_TIMEOUT)
         traces = self._parse_trace_log(self.filename)
-        self.assertEqual(len(traces), 1, "Unexpected number of traces collected")
+        expected_traces_number = 1
```

Comment:
Is a "batch" of say 512 spans considered 1 trace or 512 traces?

Reply:
and are we testing the batch size anywhere in here?

Reply:
> Is a "batch" of say 512 spans considered 1 trace or 512 traces?
"batch" can collect spans from multiple traces. Let's assume that we trace one model that can produce >= 512 spans in one request. In this case if max_batch_size is 512 and delay is long enough so that we can collect 512 traces, then when exporter exports them, they all belong to 1 trace.

In unittests, I set up the collector so that it will write trace in the file, when all spans are received, that is why I only expect 1 trace. But this trace contains multiple spans.

> and are we testing the batch size > 1 anywhere in here?
All unittests are run with default parameters for batch span processor: https://github.com/triton-inference-server/server/blob/acb5f445350b6927a7bac7b0d99b0241ccfeede7/qa/L0_trace/test.sh#L726-L733 
i.e. I don't specify them. Thus the batch size is 512 and delay 5000, so spans are exported every 5 seconds

Reply:
Makes sense, thanks 🙇‍♂️